migrations/mysql/20170209180355_add_one_off_tables_from_integration_tests/up.sql

```diff
@@ -1,5 +1,5 @@
 CREATE TABLE numbers (n INTEGER PRIMARY KEY);
 CREATE TABLE precision_numbers (n DOUBLE PRECISION NOT NULL PRIMARY KEY);
 CREATE TABLE nullable_doubles (id INT PRIMARY KEY AUTO_INCREMENT, n DOUBLE PRECISION);
-CREATE TABLE users_with_name_pk (name VARCHAR(255) PRIMARY KEY);
+CREATE TABLE users_with_name_pk (name VARCHAR(50) PRIMARY KEY);
```

Comment:
Could you explain why this change is required? It seems unrelated to the other changes.

Reply:
It has something to do with the version of MariaDB I'm using to test with (and may be mysql version 5.6).  With length 255 the migration would fail with this error https://stackoverflow.com/questions/1814532/1071-specified-key-was-too-long-max-key-length-is-767-bytes.  From looking at the diesel tests, we only need a few characters, and not 255.  I thought this would help other users testing on non-dockerized mysql.